taxi_service/settings.py

```diff
@@ -39,11 +39,13 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    "debug_toolbar",
     "taxi",
 ]
 
 MIDDLEWARE = [
     "django.middleware.security.SecurityMiddleware",
+    "debug_toolbar.middleware.DebugToolbarMiddleware",
```

Comment:
The `debug_toolbar` and its middleware should only be enabled in a development environment. Consider using an environment variable or a separate settings file to manage this configuration.